### ISSUE TITLE
TypeError: Path must be a string. Received undefined

### ISSUE THREAD

**Issue by krthr** [feathers-plus/cli#17]
Hey! When I try to generate a new service i get this error:

```
 _____                  _     _
 |  ___|   ___    __ _  | |_  | |__     ___   _ __   ___     _
 | |_     / _ \  / _` | | __| | '_ \   / _ \ | '__| / __|  _| |_
 |  _|   |  __/ | (_| | | |_  | | | | |  __/ | |    \__ \ |_   _|
 |_|      \___|  \__,_|  \__| |_| |_|  \___| |_|    |___/   |_|

Using generator version 0.4.1
Source scan took 0s 15ms
? What is the name of the service? calls

We are adding the new service calls in dir api-plus

Once this generation is complete, define the JSON-schema for the data in module
"services/calls/calls.schema.js". Then (re)generate this service.

This second generation will take the schema you added and generate
- A Mongoose model, and
- A Sequelize model, and
- Create, update and patch validation hooks.

Run "feathers-plus generate graphql" if you want any changes reflected in GraphQL.

? What is the name of the service? calls
? What would you call one row in the calls database? call
? What kind of service is it? Sequelize
? Place service code in which nested folder, e.g. `v1/blog`? (optional)
? Which path should the service be registered on? /calls
? Should this be served by GraphQL? Yes
events.js:183
      throw er; // Unhandled 'error' event
      ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at join (path.js:499:7)
    at serviceSpecsCombine (C:\Users\Workstation\AppData\Roaming\npm\node_modules\@feathers-plus\cli\node_modules\@feathers-plus\generator-feathers-plus\lib\service-specs-combine.js:11:26)
    at serviceSpecsExpand (C:\Users\Workstation\AppData\Roaming\npm\node_modules\@feathers-plus\cli\node_modules\@feathers-plus\generator-feathers-plus\lib\service-specs-expand.js:27:27)
    at generatorWriting (C:\Users\Workstation\AppData\Roaming\npm\node_modules\@feathers-plus\cli\node_modules\@feathers-plus\generator-feathers-plus\generators\writing\index.js:164:38)
    at ServiceGenerator.writing (C:\Users\Workstation\AppData\Roaming\npm\node_modules\@feathers-plus\cli\node_modules\@feathers-plus\generator-feathers-plus\generators\service\index.js:234:5)
    at Object.<anonymous> (C:\Users\Workstation\AppData\Roaming\npm\node_modules\@feathers-plus\cli\node_modules\yeoman-generator\lib\index.js:424:27)
    at C:\Users\Workstation\AppData\Roaming\npm\node_modules\@feathers-plus\cli\node_modules\run-async\index.js:25:25
    at new Promise (<anonymous>)
    at C:\Users\Workstation\AppData\Roaming\npm\node_modules\@feathers-plus\cli\node_modules\run-async\index.js:24:19
    at self.env.runLoop.add.completed (C:\Users\Workstation\AppData\Roaming\npm\node_modules\@feathers-plus\cli\node_modules\yeoman-generator\lib\index.js:425:13)
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
```

**Comment by krthr** [comment]
Node: 8.9.4
feathers-plus/cli: 0.6.1

**Comment by eddyystop** [comment]
This was fixed in f+/cli 0.6.2 using generator 0.4.2.
